app.py

```diff
@@ -287,6 +408,9 @@ def get_repository_pull_requests(repository_batch: list[dict[str, Any]]):
 
 if __name__ == "__main__":
     logger.info("Starting Bitbucket data extraction")
+    port_webhook_url = get_or_create_port_webhook()
+    if not port_webhook_url:
+        logger.error("Failed to get or create Port webhook. Skipping webhook setup...")
```

Comment:
Let's bring this to the end of the script so that users can still see data in their catalog even when the webhook creation fails.

Reply:
It is not going to terminate the runtime even if it fails but I have moved as far down as possible. The user catalog will always sync regardless of the webhook outcome